sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -588,7 +588,7 @@ private void acquireChannel(final ChannelPromiseWithExpiryTime promise) {
                     final RntbdRequestManager manager = channel.pipeline().get(RntbdRequestManager.class);
 
                     if (manager == null) {
-                        logger.warn("Channel({} --> {}) closed", channel, this.remoteAddress());
+                        logger.debug("Channel({} --> {}) closed", channel, this.remoteAddress());
```

Comment:
Same here, should we check whether the logger.isDebugEnabled?

Reply:
see above()